### PR TITLE
Add prop types to layouts

### PR DESCRIPTION
**Pull request by DmitriyKrivovyaz**

### CODE REVIEW

File: client/src/layouts/cityForecast/CurrentCityForecast/types.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types';
+
+export const locationType = PropTypes.shape({
+  name: PropTypes.string.isRequired
+});
+
+export const weatherType = PropTypes.shape({
```
Comment (ximet) [inline client/src/layouts/cityForecast/CurrentCityForecast/types.js]: move it to CurrentCityForecast.jsx

File: client/src/layouts/cityForecast/ExtraCityForecast/types.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import PropTypes from 'prop-types';
+
+export const weatherType = PropTypes.shape({
```
Comment (ximet) [inline client/src/layouts/cityForecast/ExtraCityForecast/types.js]: the same

File: client/src/App.jsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ function App(props) {
     setTheme(newTheme);
   }
 
-  useEffect(async () => {
+  useEffect(() => {
     props.initializeApp();
   }, []);
```
Comment (anton-vlasik) [inline client/src/App.jsx]: initializeApp is missing

File: client/src/types/types.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import PropTypes, { string } from 'prop-types';
+
+export const themeType = string.isRequired;
```
Comment (anton-vlasik) [inline client/src/types/types.js]: I thought we discussed that

File: client/babel.config.js [hunk 5]
```diff
@@ -17,7 +17,7 @@ module.exports = function (api) {
   ];
   const plugins = [
     '@babel/transform-react-constant-elements',
-    'transform-react-remove-prop-types',
+    // 'transform-react-remove-prop-types',
```
Comment (ximet) [inline client/babel.config.js]: remove it, if it unused

**Review by ximet — APPROVED** [review]

File: client/src/types/types.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+import PropTypes from 'prop-types';
+
+export const themeType = PropTypes.string.isRequired;
```
Comment (anton-vlasik) [inline client/src/types/types.js]: It is just a string, what is the added value for you app to have this kind of type extracted? Elaborate if you want to defend your position

**Review by anton-vlasik — APPROVED** [review]

**Pull request merged**